### PR TITLE
Add example alert for oldest_unincorporated_sct.

### PR DESCRIPTION
**Pull request by jsha**

### CODE REVIEW

File: examples/monitoring_and_alerting/prometheus/alerts/woodpecker_alerts.yml [hunk 1]
```diff
@@ -29,3 +29,10 @@ groups:
       severity: warning
     annotations:
       description: 'Errors submitting pre-certificates/certificates to log {{ $labels.uri }}'
+  - alert: OldUnincorporatedCert
+    expr: sum by (uri) (oldest_unincorporated_cert{le_log="true"}) > 7200 # 7200s = 2h
```
Comment (cpu) [inline examples/monitoring_and_alerting/prometheus/alerts/woodpecker_alerts.yml]: What is this `le_log` label? It doesn't come from `ct-woodpecker` and I suspect maybe its something SRE is injecting in the Prometheus config based on the source or log URI?

Comment (jsha) [reply examples/monitoring_and_alerting/prometheus/alerts/woodpecker_alerts.yml]: Yep, that label is injected in the Prometheus config. Fixed the example. I also chatted with SRE about ways we could tweak our alerting configs so we can maintain the same exact alert configs here as in our actual deployment, making it easier to keep them in sync.

Comment (cpu) [reply examples/monitoring_and_alerting/prometheus/alerts/woodpecker_alerts.yml]: >  I also chatted with SRE about ways we could tweak our alerting configs so we can maintain the same exact alert configs here as in our actual deployment, making it easier to keep them in sync.

Is there a ticket that summarizes this discussion and tracks the work?

Comment (jsha) [reply examples/monitoring_and_alerting/prometheus/alerts/woodpecker_alerts.yml]: Just filed one, thanks!

**Pull request merged**